robyn/__init__.py

```diff
@@ -59,6 +61,10 @@ def __init__(
         if self.config.dev:
             exit("Dev mode is not supported in the python wrapper. Please use the CLI. e.g. python3 -m robyn app.py --dev ")
 
+        if not dev_mode_env_var and os.getenv("ROBYN_DEV_MODE", False) == "True":
```

Comment:
Could you explain this line?

Reply:
here, I am checking whether the app is launched as a module or CLI. if it is CLI, the env var would be already set (before we call `load_vars` form `robyn#init` (hence, I am storing the value of `dev_mode_env_var` before calling `load_vars`.